src/App.tsx

```diff
@@ -60,7 +60,7 @@ const App: React.FC = () => {
               <Pools />
             </Route>
             <Route path="/lottery">
-              <Lottery />
+              <LotteryV2 />
```

Comment:
Should be renamed to <Lottery /> but I guess it can wait until we remove old one from repo completely

Reply:
Yup. I'll prep the OG Lottery removal PR later today